WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -132,13 +172,46 @@ class PostListFragment : Fragment(),
         dispatcher.register(this)
     }
 
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        outState.putSerializable(WordPress.SITE, site)
+        rvScrollPositionSaver.onSaveInstanceState(outState, recyclerView)
+        outState.putIntArray(KEY_TRASHED_POST_LOCAL_IDS, trashedPostIds.map { it.first }.toIntArray())
+        outState.putLongArray(KEY_TRASHED_POST_REMOTE_IDS, trashedPostIds.map { it.second }.toLongArray())
+        outState.putLongArray(KEY_UPLOADED_REMOTE_POST_IDS, uploadedPostRemoteIds.toLongArray())
+    }
+
     override fun onDestroy() {
         EventBus.getDefault().unregister(this)
         dispatcher.unregister(this)
 
         super.onDestroy()
     }
 
+    override fun onResume() {
```

Comment:
The only change in this method is the removal of `loadPosts(LoadMode.IF_CHANGED)` as it's no longer necessary with `ListStore`. It's moved for better organization, that's why it's showing as many changes. Here is the version in the base branch for easy comparison: https://github.com/wordpress-mobile/WordPress-Android/blob/feature/list-store-integration-for-post-list-master/WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt#L234-L249